backend/src/main/java/kr/momo/domain/availabledate/AvailableDates.java

```diff
@@ -47,13 +40,12 @@ public AvailableDate findByDate(LocalDate other) {
     }
 
     public boolean notExistsByDate(LocalDate other) {
-        return availableDates.stream()
-                .noneMatch(availableDate -> availableDate.isSameDate(other));
+        Meeting meeting = availableDates.first().getMeeting();
+        return !availableDates.contains(new AvailableDate(other, meeting));
     }
 
     public boolean isNotConsecutiveDay(LocalDate startDateInclusive, LocalDate endDateInclusive) {
         long availableDateRangeCount = availableDates.stream()
-                .sorted(comparing(AvailableDate::getDate))
                 .dropWhile(date -> date.isBefore(startDateInclusive))
                 .takeWhile(date -> !date.isAfter(endDateInclusive))
                 .count();
```

Comment:
```suggestion
        Meeting meeting = availableDates.first().getMeeting();
        AvailableDate from = new AvailableDate(startDateInclusive, meeting);
        AvailableDate to = new AvailableDate(endDateInclusive, meeting);
        long availableDateRangeCount = availableDates.subSet(from, to).size() + 1L;
```

`SortedSet`의 특성을 이용해서 이렇게 구현할 수도 있는데, 변경하지 않는 것이 더 깔끔한 것 같아서 수정하지 않았습니다. 🥲